stdlib/3.7/dataclasses.pyi

```diff
@@ -0,0 +1,53 @@
+from typing import overload, Any, Callable, Dict, Generic, Iterable, Mapping, Optional, Tuple, Type, TypeVar, Union
+
+
+_T = TypeVar('_T')
+
+_DictType = TypeVar('_DictType', bound=dict)
+_TupleType = TypeVar('_TupleType', bound=tuple)
+
+class _MISSING_TYPE: ...
+
+class _InitVarMeta(type): ...
+
+def asdict(obj: Any, *, dict_factory: _DictType = ...) -> _DictType: ...
+
+def astuple(obj: Any, *, tuple_factory: _TupleType = ...) -> _TupleType: ...
+
+@overload
+def dataclass(_cls: Type[_T]) -> Type[_T]: ...
+
+@overload
+def dataclass(*, init: bool = ..., repr: bool = ..., eq: bool = ...,
+  order: bool = ..., hash: Optional[bool] = ..., frozen: bool = ...) -> Callable[[Type[_T]], Type[_T]]: ...
```

Comment:
`unsafe_hash`, see https://www.python.org/dev/peps/pep-0557/#id7

Reply:
Thanks. The `hash` tristate parameter changed to `unsafe_hash` boolean in commit dbf9cff48a4ad0fd58e1c623ce1f36c3dd3d5f38.

Reply:
Fixed.